kyaml/openapi/openapi.go

```diff
@@ -65,75 +65,77 @@ const SupplementaryOpenAPIFieldName = "openAPI"
 const Definitions = "definitions"
 
 // AddSchemaFromFile reads the file at path and parses the OpenAPI definitions
-// from the field "openAPI"
-func AddSchemaFromFile(path string) error {
-	return AddSchemaFromFileUsingField(path, SupplementaryOpenAPIFieldName)
-}
+// from the field "openAPI", also returns a function to clean the added definitions
+func AddSchemaFromFile(path string) (func(), error) {
```

Comment:
Not sure what the golang code style say about this, but I somehow would expect the function to the last return value here.